paper-server/src/main/java/org/bukkit/craftbukkit/CraftServer.java

```diff
@@ -2485,6 +2486,11 @@ public Merchant createMerchant(String title) {
         return new CraftMerchantCustom(title == null ? InventoryType.MERCHANT.getDefaultTitle() : title);
     }
 
+    @Override
+    public @NotNull Merchant createMerchant() {
+        return new CraftMerchantCustom(net.kyori.adventure.text.Component.text(""));
```

Comment:
wouldn't Component.empty() be smarter?

Reply:
I'll take a look at testing what Component.empty() does when used in this scenario, but in all likelihood yes probably

Reply:
If the title will becomes nullable, there will be no need for empty component

Reply:
Titles for these menus are *not* part of the menu themselves.
They are provided by the respective menu providers, which can be *anything*.
They are not innate to the menu type itself.

A nullable title (while nice for the consumer) is a *LOT* of // diff on change work across the entire code base + hoping that won't change. I am not too convinced a nullable title makes sense here.